src/meetings.ts

```diff
@@ -114,6 +115,12 @@ export default function (zoomRequest: ReturnType<typeof request>) {
       params: params,
     })
   }
+  const ListPastMeetings = function (meetingId) {
```

Comment:
nit: thoughts on renaming to `ListPastMeetingInstances` ? (I think when reading this I thought it would return multiple meeting ids.

Reply:
makes sense!